src/avram/postgres_url.cr

```diff
@@ -1,42 +1,82 @@
 class Avram::PostgresURL
```

Comment:
Thoughts on renaming this `Avram::Credentials` since that seems to be what we're calling this now? I think the "credentials" name is a good one!

Reply:
I'm cool with that!